src/main/java/drivers/DriverFactory.java

```diff
@@ -33,6 +39,7 @@ public static WebDriver getDriver(BrowserType browser) {
                 driver = getEdge();
                 break;
             default:
+                logger.error("Please, choose a correct name of the browser...");
```

Comment:
In my opinion logger is not necessary here as in this case an exception with error message will be thrown, what do you think?